src/sentry/eventstream/kafka/dispatch.py

```diff
@@ -45,6 +45,7 @@ def dispatch_post_process_group_task(
     skip_consume: bool = False,
     group_states: GroupStates | None = None,
     occurrence_id: str | None = None,
+    eventstream_type: str | None = None,
```

Comment:
Just to confirm, no callers are passing this in yet, right? (So this is currently a no-op?)

Reply:
No callers are passing this in, this will be handled by https://github.com/getsentry/sentry/pull/79633